pkg/deployment/const.go

```diff
@@ -47,4 +47,13 @@ const (
 
 	// ConfigPaths base path for volume mounts in OpenStackAnsibleEE pod
 	ConfigPaths = "/var/lib/openstack/configs"
+
+	// RootCAIssuerInternalLabel for internal RootCA to issue internal TLS Certs
+	RootCAIssuerInternalLabel = "osp-rootca-issuer"
+
+	// RootCAIssuerInternalNamespaceLabel for internal RootCA to issue internal TLS Certs
+	RootCAIssuerInternalNamespaceLabel = "openstack"
```

Comment:
do we need this? at least right now we don't do any cross namespace actions, so the issuer will be created in the same namespace as the CR.

Reply:
removed.